lms/static/js/branding/footer.js

```diff
@@ -0,0 +1,3 @@
+function f(){
+    var a = "This is a dummy function"
+}
```

Comment:
You've added the same JS in `lms/djangoapps/branding/static/footer.js`. Why does it need to be present in both places?